src/translations/de.translation

```diff
@@ -6513,6 +6513,14 @@ Hinweis: Die Registerkarte &quot;NPSP-Einstellungen&quot; ist in der Nonprofit S
         <label>Das Accountmodell ist auf &quot;Haushalt-Account&quot; festgelegt. Daher müssen Haushalt-Regeln auf &quot;Keine Kontakte&quot; festgelegt sein. Bei der Verwendung von Haushalt-Accounts ist kein zusätzliches Haushalt-Objekt erforderlich.</label>
         <name>stgValidationHHAccountHHRules</name>
     </customLabels>
+    <customLabels>
```

Comment:
If you're going to add translations to this file, they need to be placed in the correct spot alphabetically - nothing that lowercase letters sort separately from uppercase letters. So, these two new labels should be before `<fullName>adapterException</fullName>`. This applies to all of the translation files.